src/main/java/net/sf/jabref/logic/importer/fetcher/MrDLibFetcher.java

```diff
@@ -103,17 +111,23 @@ private String makeServerRequest(String query) {
         return response;
     }
 
+    /**
+     * Constructs the query based on title of the bibentry. Adds statistical stuff to the url.
+     * @param query: the title of the bib entry.
+     * @return
+     */
     private String constructQuery(String query) {
         StringBuffer queryBuffer = new StringBuffer();
-        queryBuffer.append("https://api-dev.mr-dlib.org/v1/documents/");
-        queryBuffer.append(query);
+        //queryBuffer.append("https://api-dev.mr-dlib.org/v1/documents/");
```

Comment:
Das interface `EntryBasedFetcher` sieht die methode `getURLForID` nicht vor. Diese ist in der class `IdBasedParserFetcher` definiert. Soll ich das interface erweitern, oder einfach die methode so umbenennen?